products/appstores/new_release_ios.md

```diff
@@ -29,7 +23,8 @@ $ atom en-US/fx_ios/whatsnew/ios_6_0.lang
 
 This is the content of the new file (strings are communicated by releng).
 ```
-## NOTE: Those strings are displayed on the App Store in the what’s new section for Firefox for iOS 6.0 release. See https://l10n.mozilla-community.org//stores_l10n/locale/fr/apple/release/
+## NOTE: These strings are displayed on Google Play in the What’s new section for Firefox for iOS 6.0
```

Comment:
Indeed. And guess what? All comments in the actual files are wrong…
https://github.com/mozilla-l10n/appstores/blob/master/en-US/fx_ios/whatsnew/ios_6_0.lang

I'm tempted to let you update the files though, to build experience with the system ;-)